scripts/custom_lint_checks.py

```diff
@@ -73,17 +74,77 @@ def visit_call(self, node):
         Args:
             node. Call. The current function call node.
         """
-        if (isinstance(node.func, astroid.Name) and
+        if (isinstance(node.func, astroid.Name)) and (
                 node.func.name == self._function_name):
-            if (node.args is not None and
+            if (node.args is not None) and (
                     len(node.args) > self._positional_arguments_count):
                 self.add_message('non-explicit-kwargs', node=node)
 
 
+class HangingIndentChecker(BaseChecker):
+    """Custom pylint checker which checks for break after paren in case
+    of hanging indentation.
+    """
+    __implements__ = IRawChecker
+
+    name = 'hanging-indent'
+    priority = -1
+    msgs = {
+        'C0002': (
+            (
+                'There should be a break after parenthesis when content within '
+                'paren spans to multiple lines.'),
+            'break-after-hanging-indent',
```

Comment:
This should represent the error message. For example, the `max-len` checker displays `line-too-long` if the line exceeds 80 characters:
```bash
C:901, 0: Line too long (105/80) (line-too-long)
```
Similarly, this checker should display `no-break-after-hanging-indent` or something similar which is the actual error message.

Reply:
Done!